client/cli/src/config.rs

```diff
@@ -405,6 +405,13 @@ pub trait CliConfiguration: Sized {
 		Ok(true)
 	}
 
+	/// Get the format for the informant
+	///
+	/// By default this enables colour and does not use a prefix.
+	fn informant_output_format(&self) -> sc_informant::OutputFormat {
```

Comment:
It needs to be a Result to allow the developer who will override to put failing logic if they want.